org.eclipse.scanning.api/src/org/eclipse/scanning/api/event/queues/beans/ScanAtom.java

```diff
@@ -264,7 +264,7 @@ public boolean equals(Object obj) {
 	@Override
 	public String toString() {
 		String clazzName = this.getClass().getSimpleName();
-		
+
 		StringBuffer scanDetailsStrBuff = new StringBuffer();
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Replace the synchronized class "StringBuffer" by an unsynchronized one such as "StringBuilder". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1149)